README.md

```diff
@@ -1,18 +1,16 @@
+This action will scan the file(s) provided in the `file-with-substitutions` argument for Octopus variable substitution syntax `#{VariableName}`.  If it finds a matching variable in the `variables-file` or in the environment variables, it will replace the template with the actual value.
 
 This is a container action so it will not work on Windows runners.
 
 ## Inputs
 
-| Parameter                 | Is Required | Description                                                                                                                                                      |
-| ------------------------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `variables-file`          | true        | The json file that holds the variables and values to be used in the substitution.<br/>Defined as JSON object, e.g. `{"Variable1":"value1","Variable2":"value2"}` |
-| `file-with-substitutions` | true        | The file that has substitutions to be performed.                                                                                                                 |
-| `output-file`             | true        | The file the substitution results should be saved to.  This is generally the same value provided in `file-with-substitutions`.                                   |
+| Parameter                  | Is Required | Description                                                                                                                                                                                                                                              |
+| -------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `variables-file`           | false       | An optional yaml file containing variables to use in the substitution.                                                                                                                                                                                   |
+| `files-with-substitutions` | true        | A comma separated list of files that contains the octo tokens needing to be substituted.                                                                                                                                                                 |
+| `output-files`             | false       | An optional comma separated list of files to output.<br/>If defined, the program assumes the index of the output file is the same as the index of the template file in the template-files list. They therefore need to have the same number of elements. |
```

Comment:
I don't think I'm following this explanation, it's also referring to them as template files which the program uses but we're input it as files-with-substitutions.

Reply:
I had a hard time thinking of how to describe this behavior. Basically I want to say that if `output-files` is defined, it is assumed that each index corresponds to the matching index of the `files-with-substitutions` list.

For example if we got the following inputs
`files-with-substitutions`: ./file1.js, ./src/anotherfile.js, ./src/appsettings.config
`output-files`: ./file1.js, ./file2.js, ./appsettings.config

and were processing ./src/anotherfile.js, the outputted file would be ./file2.js. If you can think of a better way to word that, which I'm sure there is, please let me know!